prow/ANNOUNCEMENTS.md

```diff
@@ -101,6 +101,7 @@ Note: versions specified in these announcements may not include bug fixes made
 in more recent versions so it is recommended that the most recent versions are
 used when updating deployments.
 
+ - *May 18, 2020* `expiry` field has been removed from the HMAC secret.
```

Comment:
replaced with `created_at`

Reply:
Done.

Reply:
squash commits please!

Reply:
Done.
(We usually do not squash commits for Knative as developers find it harder to review new commit changes, but it seems to be a different convention for Kubernetes)